### PR TITLE
Feat: Removed old events for reactions and added !channelToggle remove

### LINKED ISSUE #498: Allow deletion of channel-toggles
`!group toggle ...` allows creating a reaction that gives per-user permissions to a certain channel.

There is nothing to remove those, meaning a Support has to manually remove all reactions and a Developer has to remove the respective entry from the database which is quite tedious.

Granted, this doesn't happen often but is inconvenient when it does.

This might also be a good time to break this thing out of the `!group` command because it has nothing to do with the pingable groups. Maybe `!channelToggle add` / `!channelToggle remove` might work (other ideas most certainly welcome)?

### PR DESCRIPTION
**Pull request by hasanlatch007**
Since `role-react` and `!group-toggle` are being refactored any events related to reactions in the old event folder can be deleted
This still needs testing and reformatting :)

!closes #498

### CODE REVIEW

**Comment by hasanlatch007** [comment]
Github why are you being shit?

**Comment by hasanlatch007** [comment]
This PR adds multiple fixes for `reactionRole`, where when adding the reaction the bot would also get the role.

File: src/programs/channel-toggle/channel-toggle-add.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import {
+  Command,
+  CommandHandler,
+  DiscordEvent,
+} from "../../event-distribution";
+import { createYesBotLogger } from "../../log";
+import prisma from "../../prisma";
+import { backfillReactions, getOrCreateMessage } from "./common";
+
+const logger = createYesBotLogger("program,", "channelToggleRemove");
```
Comment (geisterfurz007) [inline src/programs/channel-toggle/channel-toggle-add.ts:12]:
```suggestion
const logger = createYesBotLogger("program", "channelToggleRemove");
```
;)

**Review by geisterfurz007 — CHANGES_REQUESTED** [review]

**Comment by geisterfurz007** [comment]
Rest LGTM, nice work!

**Review by geisterfurz007 — APPROVED** [review]

**Pull request merged**